src/overseers/overseer.py

```diff
@@ -149,72 +195,46 @@ def get_object_by_uri(
             raise error
         return response.json()
 
-    # TODO pylint:disable=fixme
-    # we might want to add a get_objects function that let's you search for
-    # specific query param combinations. Right now it checks if the search
-    # string is in any of the object_type["target"] and "identifier" fields but often
-    # we know where those should be found, i.e. when we pass in a search_string
-    # we know it's either a pid, alternative_id or name
-    def get_objects(
-        self,
-        object_type: ObjectSearchDict,
-        search_string: str,
-    ) -> List[Dict[str, Any]]:
-        """Gets a list of objects matching the search parameters passed
-
-        This function prepares a GET request via the MyTardisRESTFactory instance and returns
-        a list of objects that match the search request.
-
-        Args:
-            object_type: A string representing the different object types that can be searched for
-            search_target: The field that is being searched on (varies from object to object)
-            search_string: The string that the search_target is being searched for.
+    def get_objects_by_identifier(
+        self, object_type: MyTardisObject, identifier: str
+    ) -> list[dict[str, Any]]:
+        """Retrieve objects from MyTardis of the given type with the given identifier"""
 
-        Returns:
-            A list of objects from the search request made.
+        if not self.type_supports_identifiers(object_type):
```

Comment:
Another duplicate of type_supports_identifiers()?

Reply:
Yeah thanks, see my reply above